models/user_context.go

```diff
@@ -0,0 +1,10 @@
+package models
+
+import "github.com/google/uuid"
+
+type UserContext struct {
+	Email          string
+	UserID         uuid.UUID `json:"sub"`
```

Comment:
Ensure that the struct tags in `UserContext` accurately reflect the data sources and usage to facilitate proper JSON serialization and deserialization. [medium]